src/lib/logical_query_plan/mock_node.hpp

```diff
@@ -24,7 +25,8 @@ class MockNode : public EnableMakeForLQPNode<MockNode>, public AbstractLQPNode {
  public:
   using ColumnDefinitions = std::vector<std::pair<DataType, std::string>>;
 
-  explicit MockNode(const ColumnDefinitions& column_definitions, const std::optional<std::string>& init_name = {});
+  explicit MockNode(const ColumnDefinitions& column_definitions, const std::optional<std::string>& init_name = {},
+                    const TableUniqueConstraints& constraints = {});
```

Comment:
Can we stick with `set_unique_constraints`? It feels more consistent to have only one way of setting these

Reply:
ok